media-libs/openexr/openexr-2.2.0-r2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-multilib
+
+DESCRIPTION="ILM's OpenEXR high dynamic-range image file format libraries"
+HOMEPAGE="http://openexr.com/"
+# changing sources. Using a revision on the binary in order
+# to keep the old one for previous ebuilds.
+SRC_URI="https://github.com/openexr/openexr/archive/v${PV}.tar.gz -> openexr-${PV}-r1.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/22" # based on SONAME
+KEYWORDS="~amd64 -arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~x86-solaris"
+IUSE="examples namespace-versioning static-libs"
+
+# Only need hard blocker for 2.2.0 series. Can remove on version bump.
+RDEPEND="sys-libs/zlib[${MULTILIB_USEDEP}]
+	 ~media-libs/ilmbase-${PV}:=[${MULTILIB_USEDEP},namespace-versioning=]
+	 !!=media-libs/ilmase-2.2.0"
```

Comment:
I had it right during testing then somehow that got deleted. I think I had the cursor in the wrong place and didn't notice.